doc/Manifest.toml

```diff
@@ -0,0 +1,83 @@
+[[Base64]]
```

Comment:
For those who wonder, the `Manifest.toml` contains the complete dependency graph of the Project and what version these are at. Documenter uses Compat which has a dependency on almost all the stdlibs which is why this file becomes a bit big.

Reply:
Does this mean that every Pkg3 package that uses Compat (i.e. almost all of them) will need to include this long list?

That seems crazy … shouldn't each package only have to list its own direct dependencies, like the REQUIRE file?

Reply:
The Manifest is not a REQUIRE file and it is not something that you will edit by hand. Instead, the Manifest is a complete record of the state of your current project. This makes it possible to e.g. send you Manifest to someone else and have them recreate the exact revisions of the packages that you had.

Reply:
Some good reading at https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html (what they call "lock"-file is our Manifest and what they call Manifest is our Project).

Reply:
If you want any real reproducibility you need to record the identity and exact version of every dependency. Otherwise how would you ever reproduce the state of a set of packages? Currently stdlib packages aren't versioned so we only record these "dangling" UUID pointers to record their identity, but in the future it will be possible to use an older stdlib package version with a newer version of Julia itself, at which point we'll need full manifest entries for stdlib packages. Of course we _won't_ need Compat for stdlibs at that point there's a tradeoff.

Reply:
Oh, it's also not necessary to check in a manifest file at all if you don't care about historical reproducibility – all you need is the project file and to then run the package version resolver.

Reply:
Yeah, the point of checking in the Manifest here is that we want to use the exact same versions all the time, to reproduce the doc build.